docs/reference/release-notes/highlights.asciidoc

```diff
@@ -106,6 +106,25 @@ search features like queries and aggregations.
 
 {es-pull}88694[#88694]
 
+[discrete]
+[[synthetic_source_technical_preview]]
+=== Synthetic `_source`
+preview::[]
+When Elasticsearch ingests documents, it creates several data structures that
+enable querying and aggregating the data. One of those data structures is called
+`_source`.  `_source` holds the fields of every document. It uses a lot of space
+and is used for specific needs, like reindexing and presenting the document in
+Kibana Discover. 
```

Comment:
You can be more precise here - something like "it's used when we need to return the exact document that you sent us, like reindex, _update, and Kibana's discover".